Src/BootCamp.Chapter/Checks.cs

```diff
@@ -17,25 +17,25 @@ public static class Checks
         public static int PromptInt(string message)
         {
             // To do: call your implementation. 
-            return 0;
+            return Lesson3.ReadInputInt(message);
         }
 
         public static string PromptString(string message)
         {
             // To do: call your implementation. 
-            return "";
+            return Lesson3.ReadInputString(message);
         }
 
         public static float PromptFloat(string message)
         {
             // To do: call your implementation. 
-            return 0;
+            return Lesson3.ReadInputFloat(message);
         }
 
         public static float CalculateBmi(float weight, float height)
         {
             // To do: call your implementation. 
-            return 0;
+            return Lesson3.CalculateBmi(height,weight);
```

Comment:
Please remove the TODO comments when you have implemented them! :)

Reply:
- removed the Todo comments in Checks
- refactored function Demo by extracting the code to the new function PersonData, to avoid duplicate code when asking the data for two persons